### ISSUE TITLE
How low is "low"?

### ISSUE THREAD

**Issue by SpencerDawkins** [moq-wg/moq-charter#33]
The current charter talks about 

> solution for a simple low-latency media delivery protocol

and then lists 

>  live streaming, gaming, and media conferencing

I'm happy to talk about what the latency requirements should be for "live streaming", "gaming", and "media conferencing", but my understanding (expressed [here for interactive/gaming](https://fiestajetsam.github.io/draft-gruessing-moq-requirements/draft-gruessing-moq-requirements.html#name-interactive-media) and [here for live streaming](https://fiestajetsam.github.io/draft-gruessing-moq-requirements/draft-gruessing-moq-requirements.html#name-live-media)) is that of these three use cases, one of these is not like the other two. 

The proposed charter should be really crisp about this, ISTM.

**Comment by SpencerDawkins** [comment]
In order to make progress, I suggest that we drop the interactive and live terms, and substitute terms that map to a numeric latency range. So, "50-100 ms (example gaming)" might be correct for what I think interactive is.

Does this help?

**Comment by fluffy** [comment]
I agree the live and interactive terms are not helping because what people want to actually do in both cases is about the same and most people think of a zoom call as being live as well as interactive. 

I think you have different idea about low than what I am seeing. Take the following fairly comment example case of a zoom call between two users in the same city in north america with good broadband connections using the browser version of zoom in chrome. This is a "pretty good" case, it gets much worse from this case. The measured one way glass-to-glass style measurement of mic to speaker was a bit over 400 ms. This is definitely interactive and is not anywhere close to 100 ms. So I think the time range used in interactive VoIP applications is probably from around 150 to 800 in my mind but is wide. Other would have different numbers but it may not matter - the important thing is we agree that 200 is better than 800. 

Gaming is even wider range depending on the type of game, delay or rollback style netcode, and many other things.

To me, the requirements are in human time, not numbers of RTT. What the applications requires is something measures in ms not number of RTT. What our solutions are capable of delivering probably is highly correlated with the varios RTTs of the users involved in the interaction to the data centers in use and RTT between the data centers.

**Comment by aboba** [comment]
Does "game streaming" refer to cloud gaming (XCloud or Stadia) or game watching (Twitch)? These actually are quite different.

**Comment by afrind** [comment]
My view is that there are two goals expressed in the opening sentence in the charter

1. Simple
2. Low latency

We want the latency to be as low as possible without (overly) compromising simplicity.  Both goals are somewhat subjective, and I think that's intentional.  The working group will need to strike a balance between them as it goes about its work.  I don't think we should put a numerical value on "low" in the charter, or promise that moq will meet every use case within a given named "clump".  The second sentence merely says moq *may* meet use cases in those clumps.
